cli/lsp/analysis.rs

```diff
@@ -479,6 +479,91 @@ impl CodeActionCollection {
     Ok(())
   }
 
+  pub(crate) fn add_deno_lint_ignore_action(
+    &mut self,
+    specifier: &ModuleSpecifier,
+    diagnostic: &lsp::Diagnostic,
+  ) -> Result<(), AnyError> {
+    let code = diagnostic
+      .code
+      .as_ref()
+      .map(|v| match v {
+        lsp::NumberOrString::String(v) => v.to_owned(),
+        _ => "".to_string(),
+      })
+      .unwrap();
+
+    let mut changes = HashMap::new();
+    HashMap::insert(
+      &mut changes,
+      specifier.clone(),
+      vec![lsp::TextEdit {
+        new_text: format!("// deno-lint-ignore {}\n", code),
+        range: lsp::Range {
+          start: lsp::Position {
+            line: diagnostic.range.start.line,
+            character: 0,
+          },
+          end: lsp::Position {
+            line: diagnostic.range.start.line,
+            character: 0,
+          },
+        },
+      }],
+    );
```

Comment:
Why not `changes.insert(specifier.clone(), vec![...])` ?

Reply:
Not sure why I used them. 😄 

Thanks!